lnwallet/wallet.go

```diff
@@ -233,13 +233,16 @@ type channelOpenMsg struct {
 // Bitcoin Core + ZeroMQ, etc. Eventually, the wallet won't require a full-node
 // at all, as SPV support is integrated inot btcwallet.
 type LightningWallet struct {
+
+	Cfg           *Config
```

Comment:
The changes in this file, other than the typo fixes should be reverted as the config changes aren't needed.